README.md

```diff
@@ -1,2 +1,20 @@
+
+## Description
+
+The repository is used to store Canadian wastewater data using the Ottawa Data Model ([ODM](https://github.com/Big-Life-Lab/ODM)).
+
+## Background
+
+Wastewater testing and surveillance (WWS) has a long history as a public health tool, helping us monitor polio outbreaks and track antimicrobial resistance. With the COVID-19 pandemic and the emergence of the SARS-CoV-2 virus, a new opportunity has emerged to leverage WWS to inform prevention and control of the pandemic. People infected with the SARS-CoV-2 virus shed the virus in their stool, meaning that the feces in wastewater systems can be checked to detect outbreaks and monitor for variants even before people become symptomatic. With over 200 wastewater testing sites across Canada, and over 2000 testing sites in over 50 countries, WWS has proven itself to be an effective tool in the fight against the virus. However, despite the rapid growth in this field and the swift deployment of WWS programs nationally, there is very little data sharing due to the absence of a centralized data repository with controlled vocabulary. This absence has led to varied data and assay quality, inconsistent reporting of wastewater test results, little adjustment of wastewater results, and has slowed the development of wastewater-based epidemiology. 
+
+That is why we developed the Canadian Open Wastewater Surveillance (COWS) Database. This centralized database, developed in tandem with the Public Health Agency of Canada and provinces, serves as an central depository for open access wastewater surveillance data. This will shrink the delay between the measurement and analysis, and provide more data for better modeling, better collaboration, and better tools in the fight against the COVID-19 pandemic. 
+
+To ensure that the COWS database is usable for all users, we have also developed a controlled vocabulary and a suite of related tools called the Open Data Model ([ODM](https://github.com/Big-Life-Lab/ODM)). The ODM an open, standard approach to share wastewater surveillance data. The ODM helps the wastewater surveillance community collaborate by allowing teams worldwide to share their data in a common structure. The model uses a relational dictionary to map more than 150 variables into 10 tables, and offers documentation on how to use the model, template files to record data in the ODM format, and scripts of code to set up a relational database according to the ODM schema.  
+
+## Current Objectives: 
+- **Create the most robust open data model for wastewater and environmental epidemiology and surveillance.** Supported by an international steering committee, the ODM dictionary will allow consistent reporting of SARS-CoV-2, including variants and mutations. The ODM will support reporting for other health hazards, including other microbes and chemicals. Reporting will expand beyond wastewater to include other environmental testing such as air and surfaces. 
```

Comment:
I know it is confusing. I suggest that we move these objectives to the ODM repository, since they are for the ODM.

It is a good idea to have COWS objectives.
- Easily share WW data. We focus on Canadian data but we are happy to receive other data.
- Allow detailed WW data (this is a distinction between COWS and other repositories - we can make reference to World Sphere and Norman).
- Easy to add data. This is where we'd mention that we use ODM tools to help validate data. We can offer to help people add data. We can automatically add your data if your data ODM format on common data platforms like Dropbox, Sync, Google Drive, ArcGIS.

Reply:
Added these goals and a mention of the "scrapping from COWS-private" structure in the latest commit - let me know if I caught everything.